include/inc_module/mod_shop/api.shop.php

```diff
@@ -240,12 +239,16 @@ function _convert_charset($string) {
 				}
 
 				if($item['shopprod_quantity']) {
+
+					//echo "<pre>";
+					//var_dump($item['shopprod_quantity']);
+					//echo "</pre>";
 					foreach($item['shopprod_quantity'] as $key => $idval) {
 
 						//loop all opt_2
-						if(!$item['shopprod_quantity'][$key]) {
-							continue;
-						}
+						//if(!$item['shopprod_quantity'][$key]) {
```

Comment:
This is git, you don't need to comment code out. If it is required again you revert the commit.